Dockerfile

```
@@ -0,0 +1,11 @@
+FROM golang:1.16 AS builder
+WORKDIR /go/src/github.com/forbole/big-dipper
+COPY . ./
+RUN CGO_ENABLED=0 go build -o /build/bdjuno ./cmd/bdjuno
+
+FROM alpine:latest
+WORKDIR /root/
+COPY --from=0 /build/bdjuno ./
+RUN ./bdjuno init
+ENTRYPOINT ["./bdjuno"]
+CMD ["parse", "--home", "/root/.bdjuno"]
```

Instead of taking care about initializing the configuration, I think it would be much better to leave the ability to run such commands to the external user. We can simply set the `CMD` to be `bdjuno` so that the user can use the container to run all the other commands (`init`, `parse`, etc) by their own:

```suggestion
CMD [ "bdjuno" ]
```